NICE.Identity.Management/ClientApp/src/components/SuggestionUser/SuggestionUser.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from "react";
+import { UserType } from "../../models/types";
+
+type SuggestionUserProps = {
+	item: UserType;
+	id: string;
+	onClick: (item: UserType) => void;
+	onMouseOver: (index: number) => void;
+};
+
+export const SuggestionUser: FC<SuggestionUserProps> = ({
+	item,
+	id,
+	onClick,
+	onMouseOver,
+}) => {
+	const { firstName, lastName, emailAddress } = item;
+	const index = Number(id.slice(id.length - 1));
+
+	const handleClick = (e: React.MouseEvent<HTMLAnchorElement>): void => {
+		e.preventDefault();
+		onClick(item);
+	};
+
+	return (
+		<a
+			id={id}
+			onClick={handleClick}
+			onMouseOver={() => onMouseOver(index)}
```

Comment:
I find the const 'index' a bit confusing here as I associate that with loops...could you just use the Number operation in here as it's not that much code?